docs/advanced.rst

```diff
@@ -102,7 +102,7 @@ C++ side, or to perform other types of customization.
 
 .. seealso::
 
-    The file :file:`example/example-operator-overloading.cpp` contains a
+    The file :file:`tests/test_operator-overloading.cpp` contains a
```

Comment:
Minor, but wouldn't it be a bit cleaner to use one convention for test filenames, dashes or underscore? I.e., `test_operator_overloading`.